sdk/core/src/services/WalletStorageService.ts

```diff
@@ -62,7 +63,9 @@ export default class WalletStorageService {
     this._clientId = options.clientId || STAGING_COGNITO_CLIENT_ID
     this._userPoolId = options.userPoolId || STAGING_COGNITO_USER_POOL_ID
 
-    const { registryUrl, issuerUrl, verifierUrl } = options
+    const { registryUrl, issuerUrl, verifierUrl, storageRegion } = options
```

Comment:
@rbrazhnyk should be a way how to call affinityWallet.saveCredentials for specific region, (not just inheirt from the SDK options)